gammapy/estimators/tests/test_utils.py

```diff
@@ -199,6 +235,49 @@ def test_compute_lightcurve_doublingtime():
     )
 
 
+def test_compute_dcf():
```

Comment:
then you use those fixtures defined above as:

```python
def test_compute_dcf(lc, lc2):
    lighcurve = lc
    lightcurve2 = lc2
    ...
```
so it's clear what inputs the test needs